index.bs

```diff
@@ -199,23 +196,30 @@ Abstract Operations {#abstract-operations}
 
 Automation {#automation}
 ==========
-This section extends the [=automation=] section defined in the Generic Sensor API [[GENERIC-SENSOR]]
-to provide mocking information about the proximity level for the purposes of
-testing a user agent's implementation of {{ProximitySensor}} API.
 
-<h3 id="mock-proximity-sensor-type">Mock Sensor Type</h3>
+This section extends [[GENERIC-SENSOR#automation]] by providing [=Proximity Sensor=]-specific virtual sensor metadata.
 
-The {{ProximitySensor}} class has an associated [=mock sensor type=] which is
-<a for="MockSensorType" enum-value>"proximity"</a>, its [=mock sensor reading values=]
-dictionary is defined as follows:
+The [=per-type virtual sensor metadata=] [=map=] must have the following [=map/entry=]:
+: [=map/key=]
+:: "`proximity`"
+: [=map/value=]
+:: A [=virtual sensor metadata=] whose [=virtual sensor metadata/virtual sensor type=] is [=Proximity Sensor=] and [=reading parsing algorithm=] is [=proximity reading parsing algorithm=].
```

Comment:
```suggestion
:: A [=virtual sensor metadata=] whose [=virtual sensor metadata/virtual sensor type=] is [=Proximity Sensor=] and [=reading parsing algorithm=] is the [=proximity reading parsing algorithm=].
```

Reply:
Fixed in https://github.com/w3c/proximity/pull/57/commits/28ddd12525270a98cd6f42f37ec1345a14eb9994